### ISSUE TITLE
Move console/engine wiki to docs

### ISSUE THREAD

**Issue by SeanKilleen** [nunit/docs#744]
Via core team discussion:

> The documentation for the console/engine exist both in the console/engine git repository AND in the docs repo, and they are not aligned.  I think @rprouse was hit by this when he did the last release, as the current doc actually was in the console repo wiki.  Can we get the docs off the console wiki and get  it all in one place, in the docs repo?

Seems like a great idea to me.

**Comment by rprouse** [comment]
I agree that the docs need to be in one place or the other, definitely not both. I am fine with either but wonder if these build docs are easier to maintain in the wiki? It isn't a strong opinion though, I've been doing PRs for the framework build and it isn't hard.
